applications/modmesh/twnia1_env.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#PBS -P <PROJECTSYSTEMCODE>
+#PBS -N build_modmesh_env
+#PBS -l select=1:ncpus=8:mpiprocs=8
+#PBS -q <NODETYPE>
+#PBS -j oe
+
+# * this script is used to build the modmesh environment
+# it should work directly in login node, or you can submit it to the queue system
```

Comment:
We should suggest users to use the queue system.  Abusing the login node is not a good way to use the cluster.